db/seeds.rb

```diff
@@ -6,3 +6,39 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```

Comment:
This Seed file has also been updated to reflect the changes made to the teacher and student models.